src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCodeV1.java

```diff
@@ -166,6 +166,8 @@ public enum ErrorCodeV1 {
 
   LEXICAL_NOT_AVAILABLE_FOR_DATABASE("Lexical search is not available on this database"),
   LEXICAL_NOT_ENABLED_FOR_COLLECTION("Lexical search is not enabled for the collection"),
+  LEXICAL_CONTENT_TOO_BIG(
+      "Lexical content is too big, please use a smaller value for the $lexical field"),
```

Comment:
Will need to figure out how to indicate maximum length constraints: 8kB value default does NOT refer to raw byte or character length of input but something like total length of all tokens indexed (which typically is somewhat smaller than original input byte length)